### DISCUSSION TITLE
footnote with popover

### DISCUSSION THREAD

**Discussion by hardin47** [quarto-dev/quarto-cli#1895] [Q&A]
Is there a way to create a popover option with a footnote?  That is, instead of having the footnote at the end of the document, I'd like it to show up when you click on the footnote.

and in case i'm not asking for enough, i'd really like the same behavior when using both html and revealjs formats.

thank you!!

**Comment by jjallaire** [comment]
I believe that this already exists but it should occur on hover not click. Are you not seeing that?

**Reply by hardin47** [reply]
right, not seeing it.  is it possibly related to this: https://www.computerhope.com/issues/ch001076.htm  ?  they say that you need both an alt text and a title?

by the way, i regularly use `fig.alt` and i used to get hover automatically.  at some point over the last year, i stopped getting any hovered text from my rendered .Rmd files.  i wonder if it is something to do with my system?

```
> sessionInfo()
R version 4.2.0 (2022-04-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Monterey 12.5

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

loaded via a namespace (and not attached):
 [1] compiler_4.2.0  fastmap_1.1.0   cli_3.3.0       htmltools_0.5.3
 [5] tools_4.2.0     rstudioapi_0.13 yaml_2.3.5      rmarkdown_2.14 
 [9] knitr_1.39      digest_0.6.29   xfun_0.31       rlang_1.0.4    
[13] evaluate_0.15  
```

**Comment by jjallaire** [comment]
It looks like we broke it! I'll investigate now.
